algo/src/test/java/com/hartwig/serve/ServeAlgoTest.java

```diff
@@ -26,6 +26,7 @@ public class ServeAlgoTest {
 
     private static final String DOCM_TSV = Resources.getResource("docm/example.tsv").getPath();
     private static final String HARTWIG_CURATED_TSV = Resources.getResource("hartwig/example.tsv").getPath();
+    private static final String HARTWIG_CURATED_GENES_TSV = Resources.getResource("hartwig/example.tsv").getPath();
```

Comment:
Suggest to structure the test resources the same as production resources (in crunch-resources-private)